README.md

```diff
@@ -65,6 +65,21 @@ Tests for this contract are implemented in Haskell using the [cleveland framewor
 For more detail about the test suite and how to execute it, check out its
 [README](./haskell/test/README.md).
 
+## Contract Metadata
+
+BaseDAO metadata should be compliant with [TZIP-16](https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-16/tzip-16.md).
+
+It can be generated via:
+
+```bash
+cd haskell
+stack exec -- baseDAO-ligo-meta print-metadata > metadata.json
+```
+
+This metadata should be stored via:
```

Comment:
`via` and the `As` (or `On`) of the following points sounds incorrect.

Also here we are giving two suggestions, but there are other valid methods.

I suggest rephrasing this as:
```suggestion
This metadata should be stored either:
```